### PR TITLE
Fix minor things

### PR DESCRIPTION
**Pull request by awlauria**

### CODE REVIEW

File: src/mca/schizo/ompi/schizo_ompi.c [hunk 1]
```diff
@@ -176,7 +176,6 @@ static struct option ompioptions[] = {
 #if PRTE_ENABLE_FT
     PMIX_OPTION_DEFINE(PRTE_CLI_ENABLE_RECOVERY, PMIX_ARG_NONE),
     PMIX_OPTION_DEFINE(PRTE_CLI_MAX_RESTARTS, PMIX_ARG_REQD),
-    PMIX_OPTION_DEFINE(PRTE_CLI_DISABLE_RECOVERY, PMIX_ARG_NONE),
```
Comment (rhc54) [inline src/mca/schizo/ompi/schizo_ompi.c:179]: Hmmm...what if the default for PRRTE has been set by MCA parameter to enable recovery, but the user doesn't want to do that for this job? Are you sure you want to remove this option?

Comment (awlauria) [reply src/mca/schizo/ompi/schizo_ompi.c:179]: It was undefined here:
https://github.com/openpmix/prrte/commit/8d735f530d6ad237d95e9e821b6ab22fbc17243a#diff-95468cad0a5594de72258b4d462586e17493568348689d12b0b612141a9312daL113

alternatively we could just add this back - oversight?

Comment (rhc54) [reply src/mca/schizo/ompi/schizo_ompi.c:179]: Yeah, that was an error on my part. I'm working right now on how to handle all these runtime controls - many of them are currently set by MCA param (e.g., the abort-on-non-zero-exit behavior), which really isn't right for PRRTE as it is primarily a DVM and not every job it runs will want such behaviors. I'm creating a new `--runtime-options` CLI for this purpose.

So we have a couple of options here:

* you could restore the definition for now, which is fine with me
* you could decide to add the `--runtime-options` directive to your cmd line and include "disablerecover" (or whatever you would like to call it) there. You'd have to wait for me to commit that directive and its supporting code - should be later today or tomorrow.

**Review by rhc54 — APPROVED** [review]

**Comment by rhc54** [comment]
bot:ibm:retest

File: src/mca/schizo/ompi/schizo_ompi.c (hunk 1 above)
Comment (awlauria) [reply src/mca/schizo/ompi/schizo_ompi.c]: Thanks - I just put it back as it's a compile error.

**Pull request merged**